### PR TITLE
Gh 9 add folsom stats

### PR DESCRIPTION
**Pull request by aerosol**

### CODE REVIEW

**Comment by aerosol** [comment]
@puzza007 @loucash @AeroNotix peep this

File: rebar.config [hunk 1]
```diff
@@ -7,7 +7,8 @@
   {'jsx', ".*", {git, "https://github.com/talentdeficit/jsx.git", {branch, "master"}}},
   {'kvlists', ".*", {git, "https://github.com/jcomellas/kvlists.git", {tag, "0.5.1"}}},
   {'proper', ".*", {git, "https://github.com/manopapad/proper", {branch, "master"}}},
-  {'lhttpc', ".*", {git, "https://github.com/esl/lhttpc", {tag, "1.3.0"}}}]}.
+  {'dlhttpc', ".*", {git, "https://github.com/ferd/dlhttpc.git", {tag, "0.2.0"}}},
+  {folsom, ".*", {git, "https://github.com/boundary/folsom.git", "master"}} ]}.
```
Comment (puzza007) [inline rebar.config]: You can save 10 bytes here by not quoting atoms :put_litter_in_its_place:

Comment (aerosol) [inline rebar.config]: :+1: :put_litter_in_its_place:

**Comment by loucash** [comment]
Awesomeness :star2:

**Comment by puzza007** [comment]
:+1:

**Pull request merged**